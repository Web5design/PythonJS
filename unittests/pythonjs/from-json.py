d = json_to_pythonjs("""{"a":1,"b":2}""")
print d["a"], d["b"]
print json_to_pythonjs("""{"a":1.2}""")["a"]
l = json_to_pythonjs("""["spam","egg"]""")
print l[0], l[1]
n = json_to_pythonjs("""{"a_list":["spam","egg"]}""")
print n["a_list"][0], n["a_list"][1]
n2 = json_to_pythonjs("""{"a":{"a":1,"b":2},"b":2}""")
print n2["a"]["a"]
