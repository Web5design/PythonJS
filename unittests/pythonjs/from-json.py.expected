1 2
1.2
spam egg
spam egg
1
